cyclus_input/predicting_the_past.xml

```diff
@@ -0,0 +1,183 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
+  <control>
+    <duration>1020</duration>
+    <startmonth>1</startmonth>
+    <startyear>1965</startyear>
+  </control>
+  <archetypes>
+    <spec><lib>cycamore</lib> <name>Enrichment</name> </spec>
+    <spec><lib>cycamore</lib> <name>Reactor</name>    </spec>
+    <spec><lib>cycamore</lib> <name>Sink</name>       </spec>
+    <spec><lib>cycamore</lib> <name>Source</name>     </spec>
+    <spec><lib>cycamore</lib> <name>Storage</name>    </spec>
+    <spec><lib>cycamore</lib> <name>DeployInst</name> </spec>
+    <spec><lib>cycamore</lib> <name>ManagerInst</name></spec>
+    <spec><lib>agents</lib>   <name>NullRegion</name> </spec>
+  </archetypes>
+  <xi:include href="./buildtimes/buildtimes.xml#xpointer(/simulation/facilities/child::*)"/>
```

Comment:
Why all these layers of tags (simulation, facilities?). The same comment I made before about these simulation tags causing confusion is true here. Please don't use extra tags if you will need to just xmlpoint through them anyway. Does that make sense? (please respond to these questions, so I know you read them)